net-misc/bird/files/ipv6-rpki-2.0.7.patch

```diff
@@ -0,0 +1,29 @@
+diff --git a/proto/rpki/transport.c b/proto/rpki/transport.c
```

Comment:
Upstream anywhere? If so, add a link at the top -- or some info about why it is needed and the source.

Reply:
The upstream commit is https://gitlab.nic.cz/labs/bird/-/commit/4e23b499696da81acf0ed5ad181573b94ccdb9a3
I can’t get back the email on the ML where we were talking about it, but it was already a known issue when the discussion came up.